src/k8s/pkg/k8sd/api/cluster_bootstrap.go

```diff
@@ -0,0 +1,56 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	apiv1 "github.com/canonical/k8s/api/v1"
+	"github.com/canonical/k8s/pkg/utils"
+	"github.com/canonical/lxd/lxd/response"
+	"github.com/canonical/microcluster/microcluster"
+	"github.com/canonical/microcluster/state"
+)
+
+func postClusterBootstrap(m *microcluster.MicroCluster, s *state.State, r *http.Request) response.Response {
```

Comment:
I don't see the `.Bootstrap` field of the request used anywhere, is there a need for it?

Reply:
I put it in because the API spec suggests we would like it? I don't see a purpose for it either... I'll remove it.